core/nodepool/config/templates/stack-template.json

```diff
@@ -12,7 +12,10 @@
   "   --volume=dns,kind=host,source=/etc/resolv.conf,readOnly=true --mount volume=dns,target=/etc/resolv.conf  \\",
   "   --volume=awsenv,kind=host,source=/var/run/coreos,readOnly=false --mount volume=awsenv,target=/var/run/coreos \\",
   "   --trust-keys-from-https \\",
-  "   {{.AWSCliImageRepo}}:{{.AWSCliTag}} -- aws s3 --region $REGION  cp {{ .S3URI }}/{{ .StackName }}/$USERDATA_FILE /var/run/coreos/",
+  {{- if .AWSCliImage.RktPullDocker }}
+  "   --insecure-options=image \\",
+  {{- end }}
+  "   {{ if .AWSCliImage.RktPullDocker }}docker://{{ end }}{{.AWSCliImage.Repo}}:{{.AWSCliTag}} --exec=aws -- s3 --region $REGION  cp {{ .S3URI }}/{{ .StackName }}/$USERDATA_FILE /var/run/coreos/",
```

Comment:
Almost the same as above at heart but how about implementing something like:
```golang
func (i Image) RktImage() string {
    if i.RktPullDocker {
        return fmt.Sprintf("docker://%s:%s", i.Repo, i.Tag)
    }
    return fmt.Sprintf("%s:%s", i.Repo, i.Tag)
}
```
and refer to it from here?
I guess we'd better to drop `*Tag`s e.g. `AWSCliTag` and integrate it to the `Image` struct to achieve this.